src/onnx/parse_convolution.cpp

```diff
@@ -149,9 +149,40 @@ struct parse_convolution : op_parser<parse_convolution>
 
         recalc_conv_attributes(values, kdims);
 
+        instruction_ref ret;
+        // parse a_zero_point and b_zero_point values
+        auto l0_zp = l0;
+        auto w_zp =  weights;
+
         op.from_value(values);
-        auto l1 = info.add_instruction(op, l0, args[1]);
-        return info.add_bias(args, l1, 1);
+        if (op.name() == "quant_convolution") {
+            if(args.size() > 2)
+            {
+                l0_zp = args[2];
+                if (l0_zp->get_shape().type() != l0_shape.type()) {
+                    MIGRAPHX_THROW("PARSE: ConvInteger Data and Data Zero Point must have same type");
+                }
+
+                l0_zp = info.add_common_op("sub", l0, l0_zp);
+
+                if(args.size() > 3)
+                {
+                    w_zp = args[3];
+                    if (w_zp->get_shape().type() != w_shape.type()) {
+                        MIGRAPHX_THROW("PARSE: ConvInteger Weight and Weight Zero Point must have same type");
+                    }
+                    
+                    w_zp = info.add_common_op("sub", weights, w_zp);
+                }
+
+                ret = info.add_instruction(op, l0_zp, w_zp);
+            }
+        }
+        else {
+            auto l1 = info.add_instruction(op, l0, args[1]);
+            ret = info.add_bias(args, l1, 1);
+        }
+        return ret;
     }
 };
 
```

Comment:
If you end up modifying this file, could you please also cut-n-paste the operator documentation from https://onnx.ai/onnx/operators/onnx__ConvInteger.html. 
Thanks.

Reply:
The problem is this parser contains code to parse both Conv and Conv Integer. I can add it to quant convolution though

Reply:
Dont paste it for quant_convolution. They are not the same operator. They are similar in some ways, but they are also different. 

I also dont see a reason to paste such documentation since it can be easily referenced.

Reply:
I agree with @pfultz2 since this is a more generic parser for convolution where functionality diverges.